target/maven-archiver/pom.properties

```diff
@@ -1,5 +0,0 @@
-#Generated by Maven
-#Tue Jul 21 14:32:08 UTC 2020
-version=1
```

Comment:
should this file be deleted?

Reply:
yea this happens to me too, do you know why target keeps making new files?

Reply:
when mvn builds it creates a .war file and other files to help it not relocate dependencies etc each time